minesweeper/auth/Auth.py

```diff
@@ -11,6 +12,7 @@ def __init__(self):
         self.current_user = None
         self.users = []
 
+        self.text_messages = text_messages
```

Comment:
Creating this variable inside each class is useless if you can use it directly from import.